src/en/contact/contact.md

```diff
@@ -22,15 +22,19 @@ With an account, use our <gcds-link external href="{{ links.githubCompsIssues }}
 
 ## Give feedback, request support, or sign up
 
+We support government employees using GC Design System. We cannot answer questions from members of the public.
+
 Use this form to provide feedback or ask questions, get help using GC Design System, or sign up for our mailing list or demo.
 
+For other enquiries, go to the [Government of Canada contacts page](https://www.canada.ca/en/contact.html) to find the department or agency that can help.
+
 <form class="my-600 contact-us-form" name="contactEN" method="post" style="min-height: 32rem;" action="/api/submission">
   <input type="hidden" name="form-name" value="contactEN" />
   <input name="honeypot" type="text" aria-label="bot" hidden/>
 
 <gcds-input type="text" name="name" input-id="name" label="Full name" size="30" autocomplete="name" required></gcds-input>
 <gcds-input type="email" name="email" input-id="email" label="Email address" size="50" autocomplete="email" required></gcds-input>
-<gcds-textarea name="message" label="Provide your feedback or ask a question if you need help" textarea-id="message"></gcds-textarea>
+<gcds-textarea name="message" label="Provide your feedback or ask a question if you need help" textarea-id="message" hint="Do not include protected or personal information."></gcds-textarea>
```

Comment:
Replace "Do not include protected or personal information" with "Avoid any personal or identifying info".